docs/pages-for-subheaders/rke1-hardening-guide.md

```diff
@@ -8,66 +8,59 @@ This document provides prescriptive guidance for how to harden an RKE cluster in
 :::
 
-This hardening guide is intended to be used for RKE clusters and associated with the following versions of the CIS Kubernetes Benchmark, Kubernetes, and Rancher:
+This hardening guide is intended to be used for RKE clusters and is associated with the following versions of the CIS Kubernetes Benchmark, Kubernetes, and Rancher:
 
 | Rancher Version | CIS Benchmark Version | Kubernetes Version           |
 |-----------------|-----------------------|------------------------------|
 | Rancher v2.7    | Benchmark v1.23       | Kubernetes v1.23 up to v1.25 |
 
-For more details on how to evaluate a hardened RKE cluster against the official CIS benchmark, refer to the RKE self-assessment guides for specific CIS benchmark versions.
+:::note
+At the time of writing, the official Kubernetes v1.25 benchmark is still in draft status by CIS. At this time Rancher is using the CIS v1.23 benchmark when scanning Kubernetes v1.25 clusters. Due to that, the CIS checks 5.2.3, 5.2.4, 5.2.5 and 5.2.6 might fail.
+:::
+
+For more details on how to evaluate a hardened RKE cluster against the official CIS benchmark, refer to the RKE self-assessment guides for specific Kubernetes and CIS benchmark versions.
 
 ## Host-level requirements
 
 ### Configure Kernel Runtime Parameters
 
-The following `sysctl` configuration is recommended for all nodes type in the cluster. Set the following parameters in `/etc/sysctl.d/90-kubelet.conf`:
+The following `sysctl` configuration is recommended for all nodes types in the cluster. Set the following parameters in `/etc/sysctl.d/90-kubelet.conf`:
 
 ```ini
 vm.overcommit_memory=1
 vm.panic_on_oom=0
 kernel.panic=10
 kernel.panic_on_oops=1
-kernel.keys.root_maxbytes=25000000
 ```
 
 Run `sysctl -p /etc/sysctl.d/90-kubelet.conf` to enable the settings.
 
 ### Configure `etcd` user and group
 
-A user account and group for the **etcd** service is required to be setup before installing RKE. 
-The **uid** and **gid** for the **etcd** user will be used in the RKE **config.yml** to set the proper permissions for files and directories during installation time.
+A user account and group for the **etcd** service is required to be set up before installing RKE.
 
 #### Create `etcd` user and group
 
-To create the **etcd** user and group run the following console commands. 
-The commands below use `52034` for **uid** and **gid** for example purposes. 
+To create the **etcd** user and group run the following console commands.
+The commands below use `52034` for **uid** and **gid** for example purposes.
 Any valid unused **uid** or **gid** could also be used in lieu of `52034`.
 
 ```bash
 groupadd --gid 52034 etcd
 useradd --comment "etcd service account" --uid 52034 --gid 52034 etcd --shell /usr/sbin/nologin
 ```
 
-Update the RKE **config.yml** with the **uid** and **gid** of the **etcd** user:
-
-```yaml
-services:
-  etcd:
-    gid: 52034
-    uid: 52034
-```
-
```

Comment:
Is this not a requirement for RKE? 
I see you keep it in the example cluster template.

Reply:
Good catch, I re-added the `## Reference Hardened RKE `cluster.yml` Configuration` section, but forgot to re-add this specific paragraph.